env.js

```diff
@@ -12,7 +19,16 @@ module.exports = {
         url: 'mongodb://127.0.0.1/aedes'
       }
     },
-    clusters: true
+    clusters: true,
+    waitForReady: true,
+    cleanDb: function (persistence, cb) {
+      clean(persistence._db, cleanopts, function (err, db) {
+        if (err) {
+          throw err
```

Comment:
You should call the callback with the error.

Reply:
Done